.github/workflows/pr-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: PR Labeler
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: WalshyDev/pr-labels@v1
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          feature: 'New Feature'
+          fix: '🐞 Bug Fix'
+          chore: 'Chore'
+          perf: '🛠️ Performance Improvement'
+          performance: '🛠️ Performance Improvement'
```

Comment:
Redundant. I think we should remove one of those, or we could approach it with a regular expression.